ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5127,3 +5127,20 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: createUserVersion3
```

Comment:
can the 2 API's createUser and createUserVersion2 be deleted?

Reply:
@SMY-ALTAMASH no they are needed

Reply:
what is the reason for this API? 
Provide a description for this API.